cli.js

```diff
@@ -41,6 +42,18 @@ var conf = pkgConf.sync('ava', {
 	}
 });
 
+// check for valid babel config shortcuts (can be either "default" or "inherit")
+var isValidShortcut = ['default', 'inherit'].indexOf(conf.babel) >= 0;
+
+if (conf.babel.length > 0 && !isValidShortcut) {
+	var message = '';
+	message += 'Value "' + conf.babel + '" in "babel" property of AVA config in package.json is invalid.';
+	message += ' Check out ' + chalk.underline('https://github.com/sindresorhus/ava#es2015-support') + ' for docs on how to configure Babel.';
```

Comment:
How about:

> Unexpected Babel configuration for AVA. See https://github.com/sindresorhus/ava#es2015-support for allowed values.

Even if users don't recall where they configured Babel, the GitHub link will make that clear. I do think we should mention the "for AVA" part to make clear we're not talking about `.babelrc` stuff.